ui/admin/app/components/form/host-set/aws/index.hbs

```diff
@@ -51,16 +51,11 @@
     {{/if}}
   </Hds::Form::Textarea::Field>
 
-  <InfoField @value={{@model.type}} disabled={{form.disabled}} as |F|>
+  <InfoField @value={{@model.type}} as |F|>
```

Comment:
Are we intending to remove `disabled={{form.disabled}}` only, but not replace?

Reply:
That is correct.